doc/nrf/ug_nrf9160.rst

```diff
@@ -199,8 +201,66 @@ Likewise, in Step 7, choose the :file:`.hex` file for the application you are pr
 .. note::
    When you update the application firmware on an nRF9160 DK, you must also update the modem firmware as described in :ref:`nrf9160_gs_updating_fw_modem`.
 
-See :ref:`gs_programming` for more information on building and programming samples on nRF9160 DK.
+.. _build_pgm_nrf9160_vsc:
+
+Building and programming using |VSC|
+====================================
+
+|vsc_extension_instructions|
+
+Complete the following steps after installing the |nRFVSC|:
+
+.. |sample_path_vsc| replace:: :file:`samples/nrf9160/aws_iot`
+
+.. |vsc_sample_board_target_line| replace:: you must use the build target ``nrf9160dk_nrf9160`` or ``nrf9160dk_nrf9160_ns`` when building the application code for the nRF9160 DK
+
+.. include:: /includes/vsc_build_and_run.txt
+
+#. Program the application:
+
+.. prog_nrf9160_start
+..
+
+   a. Set the **SW10** switch (marked debug/prog) in the **NRF91** position to program the main controller, or in the **NRF52** position to program the board controller.
+      In nRF9160 DK v0.9.0 and earlier, the switch is called **SW5**.
+   #. Connect the nRF9160 DK to your PC using a USB cable.
+   #. Power on the nRF9160 DK.
+
+.. prog_nrf9160_end
+..
+
+   d. In |VSC|, click the :guilabel:`Flash` option in the :guilabel:`Actions` panel.
+
+      If you have multiple boards connected, you are prompted to pick a device at the top of the screen.
+
+      A small notification banner appears in the bottom-right corner of |VSC| to display the progress and confirm when the flash is complete.
+
+.. _build_pgm_nrf9160_cmdline:
+
+Building and programming on the command line
+============================================
+
+.. |cmd_folder_path| replace:: on the nRF9160 DK
+
+.. |cmd_build_target| replace:: ``nrf9160dk_nrf9160`` or ``nrf9160dk_nrf9160_ns`` when building the application code for the nRF9160 DK
+
+
+.. include:: /includes/cmd_build_and_run.txt
+
+#. Program the application:
+
+.. include:: ug_nrf9160.rst
+   :start-after: prog_nrf9160_start
+   :end-before: prog_nrf9160_end
+..
+
+   d. Program the sample or application to the device using the following command:
+
+      .. code-block:: console
+
+         west flash
```

Comment:
```suggestion
         west flash --erase
```

Perhaps add a note why `--erase` is needed: The asset_tracker_v2 has secure boot enabled by default, which includes data in [One-Time Programmable area](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bootloader/README.html#otp-regions) (OTP). So to flash (including the OTP), everything has to be erased first.

Reply:
I can add a note specific for asset_tracker_v2 to use west flash --erase.

Reply:
I have added a note.